test/unit/plugin/sample_plugin.rb

```diff
@@ -0,0 +1,3 @@
+Foreman::Plugin.register :sample_plugin do
+  fixture_directory "#{Rails.root}/test/unit/plugin/fixtures/"
+end
```

Comment:
I wonder how this gets loaded, this is not ending with `_test.rb`. Can you move this to the `test_plugin_fixtures` method?

Reply:
Its loaded because of an explicit require here: https://github.com/theforeman/foreman/pull/5495/files#diff-f14ca9959ff06f3ca2ee6b9564866954R92

It has to be loaded prior to the fixture loading code gets run, and relying on it to be loaded because of *_test.rb loading did not work, it was too late.  Since this file isn't actually a test (its a plugin definition), i don't think it should end in _test.  If there's a better place to put this, i'm happy to put it somewhere else.